gnomad_qc/v3/create_release/create_hgdp_tgp_subset.py

```diff
@@ -466,6 +466,24 @@ def adjust_subset_alleles(mt: hl.MatrixTable) -> hl.MatrixTable:
     :param mt: MatrixTable to subset locus alleles
     :return: MatrixTable with alleles adjusted to only those with a sample containing a non reference allele
     """
+
+    def split_shuffle(mt: hl.MatrixTable) -> hl.MatrixTable:
+        """
+        Split rows that do and do not have the same new and old locus (which will be most) prior to the row rekey.
```

Comment:
```suggestion
        Split rows that do and do not have the same new and old locus prior to the row rekey.
        
        Most rows will have the same new and old locus annotations.
```
optional, this seemed slightly easier to read to me